roles/app/defaults/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+s3_headers: "{\"Cache-Control":"max-age=31557600\"}"
+s3_protocol: "https"
+
+attachment_url: "/spree/products/:id/:style/:basename.:extension"
+attachment_styles: "{\"mini\":[\"48x48\u003E\",\"\"],\"small\":[\"100x100\u003E\",\"\"],\"product\":[\"240x240\u003E\",\"\"],\"large\":[\"600x600\u003E\",\"\"]}"
```

Comment:
why do write `u003E` instead of `>`? I fail to see why that would be problematic

Reply:
yeah, I dont know, I just copied what was on the DB. Maybe I can change to > here and it will be converted when saving to db :+1:

Reply:
yes, let's just try and see if we can avoid some of this obfuscation.

Reply:
This does get to the DB, in DE we have > in AU we have the encoded version.
I removed this now. We can verify while testing the PR.